SU-SOE/soe_profile
# D8CORE-2617: added icon to stories paragraph type

# READY FOR REVIEW/NOT READY
- (Edit the above to reflect status)

# Summary
- Added an icon for the soe Stories paragraph type.
- This PR goes with one in the `soe_paragraphs`: https://github.com/SU-SOE/soe_paragraphs/pull/21

# Needed By (Date)
- Today, hopefully.

# Urgency
- 2 of 10

# Steps to Test

1. pull this branch.
2 `drush cim`
3. observe there is now an icon for the soe_stories paragraph.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
